ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml

```diff
@@ -5205,7 +5205,7 @@ periodics:
 - agent: kubernetes
   cluster: api.ci
   decorate: true
-  interval: 2h
+  cron: '@yearly'
```

Comment:
This seems like a big shift.  Are we sure we want so many of these periodics shifted to yearly?  Doesn't seem like it would matter much one way or the other while we care cutting so many promotion-candidate CI release images, but as that slows down, it's nice to have a floor where we will poll the jobs even if we aren't cutting new candidates (e.g. to notice provider or CI-infra issues).

Reply:
Ah, this is just for the OVN update jobs.  I'm fine with yearly for those ;)

Reply:
We are configuring the periodic runs via release-controller periodics since the `curl` section had to be removed to allow the jobs to be used as verification jobs. So they get set to `@yearly` in the job config so horologium basically just ignores them, and then we configure the actual cron/interval we want via the release-controller config (which can properly set the initial and latest image pullspecs)